src/main/java/com/three_tech_solutions/slot_app/data/mappers/PlanTypesMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.three_tech_solutions.slot_app.data.mappers;
+
+import com.three_tech_solutions.slot_app.controllers.responses.PlanTypesResponse;
+import com.three_tech_solutions.slot_app.data.enums.PlanType;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class PlanTypesMapper {
+
+    public static PlanTypesResponse toResponse(PlanType planType) {
+        return new PlanTypesResponse(planType.getName());
+    }
+
+    public static List<PlanTypesResponse> toResponseList() {
+        return Arrays.stream(PlanType.values())
+                .map(PlanTypesMapper::toResponse)
+                .toList();
+    }
```

Comment:
Aca el mapper, es como que estuviese teniendo la responsabilidad de obtener los valores de los tipos de plan, cuando deberia recibirlo como parametro


```suggestion
    public static List<PlanTypesResponse> toResponseList(List<PlanType> planTypes) {
        return planTypes
                .stream()
                .map(PlanTypesMapper::toResponse)
                .toList();
    }
```